src/components/SignTransactionWallets/index.ts

```diff
@@ -19,9 +16,9 @@ export const WALLET_STEPS: SigningComponents = {
   [WalletId.LEDGER_NANO_S_NEW]: SignTransactionLedger,
   [WalletId.TREZOR]: SignTransactionTrezor,
   [WalletId.TREZOR_NEW]: SignTransactionTrezor,
-  [WalletId.KEYSTORE_FILE]: SignTransactionKeystore,
-  [WalletId.MNEMONIC_PHRASE]: SignTransactionMnemonic,
-  [WalletId.MNEMONIC_PHRASE_NEW]: SignTransactionMnemonic,
+  [WalletId.KEYSTORE_FILE]: null,
+  [WalletId.MNEMONIC_PHRASE]: null,
+  [WalletId.MNEMONIC_PHRASE_NEW]: null,
```

Comment:
Any reason to preserve these wallet ids ?

Reply:
I wasn't sure what the exact ramifications of that would be.